artwork/logo/coala-header.svg

```diff
@@ -10,28 +10,89 @@
    xmlns:xlink="http://www.w3.org/1999/xlink"
    xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
    xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
-   viewBox="0 0 193.2 41.099998"
-   version="1.1"
+   x="0px"
+   y="0px"
+   viewBox="0 0 193.2 41.1"
```

Comment:
these changes should be done in the other commit.

I doubt this is part of fixing coala.io, but it is a sensible change as part of fixing the file structure (the other commit).
```diff
-viewBox="0 0 193.2 41.099998"
+viewBox="0 0 193.2 41.1
```